packages/table/test/virtualized-table-selects.test.ts

```diff
@@ -458,9 +460,13 @@ describe('Virtualized Table Selects', () => {
             `
         );
         const el = test.querySelector('sp-table') as Table;
+        const body = el.querySelector(
+            'sp-table-body'
+        ) as unknown as TableBody & {
+            [virtualizerRef]: Virtualizer;
+        };
 
-        await oneEvent(el, 'rangeChanged');
-        await elementUpdated(el);
+        await body[virtualizerRef].layoutComplete;
```

Comment:
Should we make this:
```
await tableLayoutComplete(el);
```

And then have a helper at the top or in a different file like:

```
async function tableLayoutComplete(table: Table): Promise<void> {
    const body = el.querySelector(
            'sp-table-body'
        ) as unknown as TableBody & {
            [virtualizerRef]: Virtualizer;
        };
    await body[virtualizerRef].layoutComplete; 
}
```

Can accept this as follow up work if you'll link the issue you create here.

Reply:
Nah, I can do it! Thank you!